src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperDescriptorResolver.cs

```diff
@@ -26,14 +26,16 @@ public class TagHelperDescriptorResolver : ITagHelperDescriptorResolver
             };
 
         private readonly TagHelperTypeResolver _typeResolver;
+        private readonly bool _designTime;
 
         /// <summary>
         /// Instantiates a new instance of the <see cref="TagHelperDescriptorResolver"/> class.
         /// </summary>
-        public TagHelperDescriptorResolver()
-            : this(new TagHelperTypeResolver())
+        /// <param name="designTime">Indicates if the <see cref="TagHelperDescriptorResolver"/> should run in
+        /// design time mode</param>
```

Comment:
Be more specific, similar to `TagHelperDescriptorFactory` doc comments e.g. `Indicates whether resolved <see cref="TagHelperDescriptor"/>s should include design time-specific information`.